calendar-app/Components/GroupSection/InGroupSection.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from "react";
+import { Alert, View, StyleSheet, Text, FlatList } from "react-native";
+import TextInputBar from "../TextInputBar";
+import ButtonComp from '../ButtonComp';
+import { loginUserAPI } from '../API/Users/UsernameCheck';
+import { inviteToGroup } from '../API/Invites/InviteToGroup';
+import { getUserId } from "../Storage/userDataStorage";
+import { GlobalColor, GlobalFont, GlobalSecondaryColor, GlobalTextColor } from '../../Styles';
+
+const GroupInput = () => {
+  const [username, setUsername] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [user, setUser] = useState(null);
+  const [members, setMembers] = useState([]);
+
+
+  const handleSendInvite = async () => {
+    if (!username) {
+      Alert.alert('Please enter a username');
+      return;
+    }
+    
+    try {
+      const fetchedUser = await loginUserAPI({ username });
+      if (fetchedUser && fetchedUser.group_id) {
+        Alert.alert('User already in a group');
+        return;
+      }
+      
+      if (fetchedUser && !fetchedUser.group_id) {
+        const group_id = await getUserId();
+        await inviteToGroup({ user_id: fetchedUser.id, group_id });
```

Comment:
this should be await inviteToGroup({ user_id: fetchedUser.id, group_id: fetchedUser.group_id )}; Cause an interface is created for this methd but only user id is passed to it